[BUG] Cannot build: "local import in non-local package"

**BUG-FILTER-CHECKLIST.** Anyone who opens up a bug report must fill these checkboxes (add an "x" to `[ ]` to be like `[x]`) to assume that he acknowledged that he red README.md and followed checklist below.
- [x] I'm using vanilla game and acknowledged about bugs and its workarounds.
- [x] I have direct access to shell that runs FactoCord. (via tmux, screen or whatever that pipes `stdin` **and** `stderr` to output)

**Describe the bug**
Cannot build FactoCord-3.0 with current version of Go (1.14), both without and with a `go.mod` file.

**To Reproduce**

```
$ go version
go version go1.14.4 darwin/amd64

$ go build ./...
go: cannot find main module, but found .git/config in /home/jameslucktaylor/git/github.com/maxsupermanhd/FactoCord-3.0
        to create a module there, run:
        go mod init

$ go mod init github.com/maxsupermanhd/FactoCord-3.0
go: creating new go.mod: module github.com/maxsupermanhd/FactoCord-3.0

$ go build ./...
main.go:12:2: local import "./commands/admin" in non-local package
main.go:13:2: local import "./discord" in non-local package
main.go:14:2: local import "./support" in non-local package
commands/commands.go:7:2: local import "./admin" in non-local package
commands/commands.go:8:2: local import "./utils" in non-local package
commands/commands.go:6:2: local import "../support" in non-local package
commands/admin/ban.go:6:2: local import "../../support" in non-local package
commands/utils/mods.go:8:2: local import "../../support" in non-local package
discord/chat.go:11:2: local import "../commands" in non-local package
discord/cache.go:8:2: local import "../support" in non-local package
```

**Expected behavior**
Expect `go build` to produce a working binary, and `go build ./...` to exit with a status of zero and without any errors.

**My instance information**
- OS: `macOS Mojave 10.14.6 (18G5033)`
- golang version: `go version go1.14.4 darwin/amd64`
- FactoCord version: `6b61a42f866b2a3862735b6a66ffece57234cc7b`
- Factorio version: N/A
- Server config: N/A
- Factorio and bot logs: N/A
- Run conditions: N/A

**Screenshots**

--- a/commands/admin/ban.go
+++ b/commands/admin/ban.go
@@ -3,8 +3,9 @@ package admin
 import (
 	"strings"
 
-	"../../support"
 	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // BanPlayerUsage comment...

--- a/commands/admin/kick.go
+++ b/commands/admin/kick.go
@@ -3,8 +3,9 @@ package admin
 import (
 	"strings"
 
-	"../../support"
 	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // KickPlayerUsage comment...

--- a/commands/admin/mod.go
+++ b/commands/admin/mod.go
@@ -6,8 +6,9 @@ import (
 	"io/ioutil"
 	"strings"
 
-	"../../support"
 	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // ModJSON is struct containing a slice of Mod.

--- a/commands/admin/restart.go
+++ b/commands/admin/restart.go
@@ -3,8 +3,9 @@ package admin
 import (
 	"time"
 
-	"../../support"
 	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // R is a reference to the running boolean in main.

--- a/commands/admin/save.go
+++ b/commands/admin/save.go
@@ -1,8 +1,9 @@
 package admin
 
 import (
-	"../../support"
 	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // SaveServer executes the save command on the server.

--- a/commands/admin/stop.go
+++ b/commands/admin/stop.go
@@ -1,8 +1,9 @@
 package admin
 
 import (
-	"../../support"
 	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // StopServer saves and stops the server.

--- a/commands/admin/unban.go
+++ b/commands/admin/unban.go
@@ -3,8 +3,9 @@ package admin
 import (
 	"strings"
 
-	"../../support"
 	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // UnbanPlayerUsage comment...

--- a/commands/commands.go
+++ b/commands/commands.go
@@ -3,10 +3,11 @@ package commands
 import (
 	"strings"
 
-	"../support"
-	"./admin"
-	"./utils"
 	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/commands/admin"
+	"github.com/maxsupermanhd/FactoCord-3.0/commands/utils"
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // Command is a struct containing fields that hold command information.

--- a/commands/utils/mods.go
+++ b/commands/utils/mods.go
@@ -5,8 +5,9 @@ import (
 	"fmt"
 	"io/ioutil"
 
-	"../../support"
 	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // ModJson is struct containing a slice of Mod.

--- a/commands/utils/version.go
+++ b/commands/utils/version.go
@@ -4,8 +4,9 @@ import (
 	"os/exec"
 	"strings"
 
-	"../../support"
 	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 var VersionStringUsage = "Usage: $version"

--- a/discord/cache.go
+++ b/discord/cache.go
@@ -1,11 +1,12 @@
 package discord
 
 import (
-	"github.com/bwmarrin/discordgo"
 	"strings"
 	"time"
 
-	"../support"
+	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 func CacheUpdater(session *discordgo.Session) {

--- a/discord/chat.go
+++ b/discord/chat.go
@@ -8,8 +8,8 @@ import (
 
 	"github.com/bwmarrin/discordgo"
 
-	"../commands"
-	"../support"
+	"github.com/maxsupermanhd/FactoCord-3.0/commands"
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // fuck golang. it's shit

--- a/discord/log.go
+++ b/discord/log.go
@@ -2,11 +2,12 @@ package discord
 
 import (
 	"encoding/json"
-	"github.com/bwmarrin/discordgo"
 	"regexp"
 	"strings"
 
-	"../support"
+	"github.com/bwmarrin/discordgo"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 type FactorioLogWatcher struct {

--- a/go.mod
+++ b/go.mod
@@ -1,0 +1,8 @@
+module github.com/maxsupermanhd/FactoCord-3.0
+
+go 1.14
+
+require (
+	github.com/bwmarrin/discordgo v0.21.1
+	github.com/joho/godotenv v1.3.0
+)

--- a/main.go
+++ b/main.go
@@ -9,10 +9,11 @@ import (
 	"os/signal"
 	"time"
 
-	"./commands/admin"
-	"./discord"
-	"./support"
 	_ "github.com/joho/godotenv/autoload"
+
+	"github.com/maxsupermanhd/FactoCord-3.0/commands/admin"
+	"github.com/maxsupermanhd/FactoCord-3.0/discord"
+	"github.com/maxsupermanhd/FactoCord-3.0/support"
 )
 
 // Running is the boolean that tells if the server is running or not

